scripts/install.sh

```diff
@@ -27,10 +32,23 @@ printf "\n======== Applying patches ========\n"
         exit 1
 }
 
+
+# Generate Licenses
+printf "\n======== Generate Licenses ========\n"
+cd ${PROJ_ROOT}/vscode
+cp LICENSE.txt LICENSE.vscode.txt
+cp ThirdPartyNotices.txt LICENSE-THIRD-PARTY.vscode.txt
+cp ../LICENSE-THIRD-PARTY .
+cd ${PROJ_ROOT}
+
 # Comment out breaking lines in postinstall.js
 printf "\n======== Comment out breaking git config lines in postinstall.js ========\n"
 sh ${PROJ_ROOT}/scripts/postinstall.sh
 
+# Build tarball for conda feedstock from vscode dir
+printf "\n======== Build Tarball for Conda Feedstock ========\n"
+bash ${PROJ_ROOT}/scripts/create_code_editor_tarball.sh -v 1.2.0
```

Comment:
Can we make the version number an input instead of hard coded

Reply:
yup I was thinking of the same. 
I was thinking we should have a version file and use that to take version value at all places